presto/dbapi.py

```diff
@@ -281,12 +281,24 @@ def _prepare_statement(self, operation, statement_name):
 
         raise presto.exceptions.FailedToObtainAddedPrepareHeader
 
+    def _format_prepared_param(self, param):
+        """
+        Formats parameters to be passed in an
+        EXECUTE statement.
+        """
+        if isinstance(param, str):
+            # escape single quotes to protect against
+            # SQL injection in the execute
+            return ("'%s'" % param.replace("'", "''"))
+        else:
+            return str(param)
```

Comment:
what else type `param` could be?

since we support python 2.7 still, this could be eg unicode.

also, there are many other cases which would need to be handled correctly (maps, lists, floating point infinity or NaN, maybe others), so please file an issue and add TODO here (with a link)

Reply:
agree, I tagged https://github.com/prestosql/presto-python-client/issues/45 as the issue for this. I will try to follow up with another MR to address this.

Reply:
tried to address this in #51